presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestMemoBasedLookup.java

```diff
@@ -153,12 +152,12 @@ public PlanNodeStatsEstimate calculateStats(
             double outputRows = 1;
             for (PlanNode source : planNode.getSources()) {
                 PlanNodeStatsEstimate sourceStats = lookup.getStats(source, session, types);
-                if (!sourceStats.getOutputRowCount().isValueUnknown()) {
-                    outputRows += sourceStats.getOutputRowCount().getValue();
+                if (!Double.isNaN(sourceStats.getOutputRowCount())) {
+                    outputRows += sourceStats.getOutputRowCount();
```

Comment:
This is test only, so maybe doesn't matter much, but I don't get why explicitly filter out NaNs.